portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Becca's Portfolio</h1>
+        <p>Welcome!</p>
+        <p>I'm a rising junior computer science major at Vanderbilt University</p>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <p>Click here to get a random fun fact about me:</p>
       <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
Maybe change the text on the button to something other than "Hello" since it's no longer a greeting?

Reply:
done!

Reply:
done!